apps/src/code-studio/initializeBlockPreview.js

```diff
@@ -17,7 +17,9 @@ function update(blockSpace, container, editor) {
   var metrics = blockSpace.getMetrics();
   var height = metrics.contentHeight + metrics.contentTop;
   container.style.height = height + 'px';
-  blockSpace.blockSpaceEditor.svgResize();
+  if (Blockly.version === 'CDO') {
```

Comment:
Calling on `Blockly.svgResize(blockSpace)` if the Blockly version is `Google` truncates the blocks when they're displayed.

Reply:
While pair reviewing, we agreed that we should be calling `svgResize` for either Blockly version so that the workspace svg fills its container. We learned that blocks were getting cut off because the container itself was too narrow.

Reply:
We discovered that the container in question is the `span4` element here: 
https://github.com/code-dot-org/code-dot-org/blob/alice/google-blockly-level-edit/dashboard/app/views/levels/editors/fields/_collapsible_block_editor.html.haml#L7-L9

Using a wider span class would allow most blocks to fit in the workspace svg after it was resized. 

Separately, I recall that years ago these two elements were aligned side-by-side. The combined width of a `span8` and `span4` element add confidence to that recollection as they would fit comfortably together within their container element. For this PR, it probably makes sense to just expand the latter, but as a follow-up we could consider restoring the older alignment. (Doing so would re-introduce cropping of our largest blocks so we'd want to confirm which option is preferred by our levelbuilder audience.)

Reply:
Thanks for pairing with me on this! I agree that it would be nice if the xml and blocks preview were side-by-side. Looking forward to seeing what curriculum folks think.